src/main.jsx

```diff
@@ -2,9 +2,12 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import './index.css'
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
Since this app is freshly created please consider using the latest APIs from the `react-router` library.
Refer: https://reactrouter.com/en/main/routers/picking-a-router#using-v64-data-apis